src/Symfony/Component/Translation/DependencyInjection/TranslatorPass.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Translation\DependencyInjection;
+
+use Symfony\Component\DependencyInjection\Reference;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\Compiler\ServiceLocatorTagPass;
+
+class TranslatorPass implements CompilerPassInterface
+{
+    /**
+     * @var string
+     */
+    private $translatorServiceId;
+
+    /**
+     * @var string
+     */
+    private $loaderServiceId;
+
+    /**
+     * @var string
+     */
+    private $loaderTag;
+
+    /**
+     * @param string $translatorServiceId
+     * @param string $loaderServiceId
+     * @param string $loaderTag
+     */
```

Comment:
phpdoc should be removed (same for properties, applies on the 3 passes), it doesn't give anything that an IDE couldn't guess